WordPress/Classes/Models/BlockEditorSettingElement+CoreDataProperties.swift

```diff
@@ -11,12 +11,6 @@ enum BlockEditorSettingElementTypes: String {
     }
 }
 
-enum BlockEditorExperimentalFeatureKeys: String {
-    case galleryWithImageBlocks
-    case quoteBlockV2
-    case listBlockV2
```

Comment:
Were the quote and list values already unused by the editor?

Reply:
Correct! I found one remaining [reference](https://github.com/WordPress/gutenberg/blob/73719c4281e49548d3ba9456a8a0064967cd4875/packages/react-native-editor/ios/GutenbergDemo/GutenbergViewController.swift#L373-L374) that I'll remove and link to this PR so we can include it as well.